js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -33,10 +34,15 @@ const IS_LEAF = `${NODE_CLASS}-is-leaf`;
 
 const ITEM_CLASS = `${WIDGET_CLASS}-item`;
 const ITEM_WITH_CHECKBOX_CLASS = `${ITEM_CLASS}-with-checkbox`;
+const ITEM_WITH_CUSTOM_EXPANDER_ICON_CLASS = `${ITEM_CLASS}-with-custom-expander-icon`;
 const ITEM_WITHOUT_CHECKBOX_CLASS = `${ITEM_CLASS}-without-checkbox`;
 const ITEM_DATA_KEY = `${ITEM_CLASS}-data`;
 
 const TOGGLE_ITEM_VISIBILITY_CLASS = `${WIDGET_CLASS}-toggle-item-visibility`;
+const CUSTOM_ICON_TOGGLE_ITEM_VISIBILITY_CLASS = `${WIDGET_CLASS}-custom-icon-toggle-item-visibility`;
+const CUSTOM_EXPAND_ICON_CLASS = `${CUSTOM_ICON_TOGGLE_ITEM_VISIBILITY_CLASS}-expand-button-icon`;
```

Comment:
This names are too long)
I also didn't catch y we need to add this classes because we didn't add such classes for usual icons

Reply:
i added these classes to manipulate them in code, i used TOGGLE_ITEM_VISIBILITY_CLASS name to be consistent with already existing class name. if you want to shorten them i guess no problem